uber/custom_tags.py

```diff
@@ -508,14 +508,13 @@ def _notice(self, label, takedown, amount_extra):
         else:
             for day, price in sorted(c.PRICE_BUMPS.items()):
                 if day < takedown and localized_now() < day:
```

Comment:
As you can see, we just use a different CSS class for "price goes up" notices than we do for "prereg closes" notices, and style them the same way so that we can show/hide the appropriate notice type.